src/main/java/lcduke/TaskList.java

```diff
@@ -0,0 +1,182 @@
+package lcduke;
+
+import java.text.ParseException;
+import static java.lang.Integer.parseInt;
+
+/** Ths creates a TaskList object.
+ */
+
+public class TaskList {
+    static Task[] totalTasks = new Task[100];
```

Comment:
Since there is no limit to number of tasks, I think it is better to use array list instead of array. In case you need to expand the project to be able to hold more than 100 tasks, you will need to modify this piece of code repetedly.